### ISSUE TITLE
Crash on startup - theme parsing error

### ISSUE THREAD

**Issue by OttoRobba** [howl-editor/howl#270]
Using Arch Linux, with latest Gnome (3.22), Moonscript (0.5) and Howl (0.4.1), I get the following error output when trying to run Howl:

```
(howl:1110): Gtk-WARNING **: Theme parsing error: <data>:11:39: Using Pango syntax for the font: style property is deprecated; please use CSS syntax

(howl:1110): Gtk-WARNING **: Theme parsing error: <data>:16:39: Using Pango syntax for the font: style property is deprecated; please use CSS syntax

(howl:1110): Gtk-WARNING **: Theme parsing error: <data>:20:46: Using Pango syntax for the font: style property is deprecated; please use CSS syntax

(howl:1110): Gtk-WARNING **: Theme parsing error: <data>:23:65: Using Pango syntax for the font: style property is deprecated; please use CSS syntax

(howl:1110): Gtk-WARNING **: Theme parsing error: <data>:24:49: Using Pango syntax for the font: style property is deprecated; please use CSS syntax

(howl:1110): Gtk-WARNING **: Theme parsing error: <data>:25:52: Using Pango syntax for the font: style property is deprecated; please use CSS syntax

(howl:1110): Gtk-WARNING **: Theme parsing error: <data>:27:46: Using Pango syntax for the font: style property is deprecated; please use CSS syntax

(howl:1110): Gtk-WARNING **: Theme parsing error: <data>:31:46: Using Pango syntax for the font: style property is deprecated; please use CSS syntax

(howl:1110): Gtk-WARNING **: Theme parsing error: <data>:35:46: Using Pango syntax for the font: style property is deprecated; please use CSS syntax
Failed to dispatch 'signal activate: ../lib/ljglibs/gtk/css_provider.moon:12: <data>:11:8not a number (code: 1)'
```

**Comment by OttoRobba** [comment]
Seems to be fixed in git version. Built from source, Howl opens normally.

**Comment by nilnor** [comment]
FYI: I just released 0.4.1 as well now, which contains this fix as well.
